src/Makefile.am

```diff
@@ -34,6 +34,10 @@ endif
 
 EXTRA_LIBRARIES =
 
+AM_CXXFLAGS += -O0 -g
+AM_CPPFLAGS += -O0 -g
+AM_LDFLAGS  += -O0 -g
```

Comment:
Those changes are unrelated to the QA python framework.

Reply:
Maybe the timeout we are seeing on travis for builds that usually take a lot less time are due to this change.

E.g. in the following build you see how we don't even reach the end of bitcoin-qt compilation whereas for this conf hte compilation time is around 10-15 mins.

https://travis-ci.org/BitcoinUnlimited/BitcoinUnlimited/jobs/593114808

Reply:
nice catch!  This was added temporarily for easier debugging